test/integration/acceptance/custom/basic/basic.go

```diff
@@ -68,16 +70,46 @@ func (r *BasicTestRunner) Setup(ctx context.Context, createOptsPublic types.Site
 	err = prv1Cluster.VanClient.RouterCreate(ctx, *siteConfig)
 	assert.Assert(t, err)
 
+	var podStartTimeBefore *v1.Time
+	if testSync == true {
+		// Pick the pod details for config-sync validation
+		pods, _ := kube.GetPods("skupper.io/component=router", prv1Cluster.Namespace, prv1Cluster.VanClient.KubeClient)
+		assert.Assert(t, len(pods) > 0)
+		podStartTimeBefore = pods[0].Status.StartTime
+	}
+
 	var connectorCreateOpts types.ConnectorCreateOptions = types.ConnectorCreateOptions{
 		SkupperNamespace: prv1Cluster.Namespace,
 		Name:             "",
 		Cost:             0,
 	}
 	_, err = prv1Cluster.VanClient.ConnectorCreateFromFile(ctx, secretFile, connectorCreateOpts)
 	assert.Assert(t, err)
+
+	var podStartTimeAfter *v1.Time
+	if testSync == true {
+		// Wait for connection completion
+		time.Sleep(5 * time.Second)
+
+		cli, _ := client.NewClient("private-basic-1", "", "")
+		linkStatus, err := cli.ConnectorInspect(context.Background(), "link1")
+		assert.Assert(t, err)
+		if err != nil {
+			log.Fatalf("[TestSync] error: %v", err)
+			return
+		}
+		assert.Assert(t, linkStatus.Connected == true)
+
+		pods, _ := kube.GetPods("skupper.io/component=router", prv1Cluster.Namespace, prv1Cluster.VanClient.KubeClient)
+		assert.Assert(t, len(pods) > 0)
+		podStartTimeAfter = pods[0].Status.StartTime
+
+		// Check if the container has restarted
+		assert.Assert(t, podStartTimeAfter.Equal(podStartTimeBefore))
+	}
 }
 
-func (r *BasicTestRunner) TearDown(ctx context.Context) {
```

Comment:
Why did you remove the context?

Reply:
It was not in use, so I removed it. But I am setting it back, to keep the same signature for TearDown